applications/asset_tracker_v2/src/cloud/aws_iot_integration.c

```diff
@@ -283,7 +283,7 @@ int cloud_wrap_init(cloud_wrap_evt_handler_t event_handler)
 	char imei_buf[20];
```

Comment:
I think the buffer size needs to be increased as the response contains "OK\r\n" now that nrf_modem_at is used

Reply:
Thanks, should have fixed this now. There are a few issues when building the project for `native_posix` and `qemu_cortex_m3`. Do you have some ideas on how to fix those?